api/client/commands.go

```diff
@@ -2722,6 +2732,7 @@ func (s *containerStats) Display(w io.Writer) error {
 
 func (cli *DockerCli) CmdStats(args ...string) error {
 	cmd := cli.Subcmd("stats", "CONTAINER [CONTAINER...]", "Display a live stream of one or more containers' resource usage statistics", true)
+	noStream := cmd.Bool([]string{"-no-stream"}, false, "Disable streaming stats and only pull the first result")
```

Comment:
Because it would have to be defaulted to true, and as such someone would have to type out the full `--stream=false`, instead of simply `--no-stream`